kotlin-android/app/src/main/java/com/mongodb/tasktracker/model/Member.kt

```diff
@@ -0,0 +1,8 @@
+package com.mongodb.tasktracker.model
+
+import org.bson.Document
+
+class Member (document: Document) {
+    val id : String = document["_id"] as String
+    val name : String = document["name"] as String
+}
```

Comment:
nit: newline at the end of the file (applies to several pages)

Reply:
Will double-check this across the whole repo before I merge!